packages/core/src/elements/Button/FeButton.tsx

```diff
@@ -2,23 +2,36 @@ import React, { forwardRef } from 'react';
 import classNames from 'classnames';
 import './FeButton.scss';
 import { ButtonProps } from './interfaces';
+import { FeLoader } from '../Loader';
 
 const prefixCls = 'fe-button';
 export const FeButton = forwardRef<HTMLButtonElement, ButtonProps>((props, ref) => {
-  const { className, children, variant, size, disabled, loading, fullWidth, ...restProps } = props;
+  const {
+    className,
+    children,
+    variant,
+    size,
+    disabled: disabledStateFromProps,
+    loading,
+    fullWidth,
+    ...restProps
+  } = props;
   const { isCancel, inForm, asLink, ...propsWithoutJunk } = restProps;
 
+  const isButtonDisabled = disabledStateFromProps || loading;
```

Comment:
remove the disable property from the destruction, and use it like:

`props.disabled`
`const disabled = props.disabled || loading`.

HTMLButtonElement should receive the disable as props also.